stable/kommander/Chart.yaml

```
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: 1.1.0
+appVersion: 1.1.2
```

```suggestion
appVersion: 1.1.0
```
this doesn't change.